app/presenters/vendor_api/single_application_presenter.rb

```diff
@@ -369,8 +369,26 @@ def hesa_itt_data
           sex: equality_and_diversity_data['hesa_sex'],
           disability: equality_and_diversity_data['hesa_disabilities'],
           ethnicity: equality_and_diversity_data['hesa_ethnicity'],
-        }
+        }.merge(additional_hesa_itt_data(equality_and_diversity_data))
+      end
+    end
+
+    def additional_hesa_itt_data(equality_and_diversity_data)
+      standard_disabilities = DisabilityHelper::STANDARD_DISABILITIES.map(&:last)
+      other_disability_text = (equality_and_diversity_data['disabilities'] - standard_disabilities).first.presence
+
+      other_ethnicity_code = Hesa::Ethnicity.find('Another ethnic background', application_form.recruitment_cycle_year)&.hesa_code
```

Comment:
There are other ways of saying a candidate has "Other" ethnicity, but they are a bit more specific like "Other white background" & they have different codes, so I've left them alone here

Reply:
I think we need to account for any text entered by the user, which includes any "other ..." field

Reply:
We also need to ignore the standard text which could look like "Another Mixed background" which is not entered by the user